src/store/composer.json

```diff
@@ -32,7 +32,7 @@
         }
     ],
     "require": {
-        "php": ">=8.2",
+        "php": ">=8.4",
```

Comment:
is there are reason besides those property hooks for bumping the php version?

in general i don't think we should do that

Reply:
Decided against attributes because these objects are passed through a bunch of services. Just went with old-school interfaces instead.